### ISSUE TITLE
Breaking in FW 5.14.0

### ISSUE THREAD

**Issue by chrisgillison** [cgkineo/adapt-articleBlockSlider#54]
Appreciate this extension is a work in progress but for the record, FW 5 version of it worked great in 5.6.1. I jumped to FW 5.14.0 and it breaks.

In my console I'm getting 2 errors:

Uncaught TypeError: firstQuestionBlock is undefined in adapt.min.js

and

Source map error: Error: request failed with status 404
Resource URL: http://localhost:9001/libraries/regenerator-runtime.min.js

**Comment by tomgreenfield** [comment]
Could you rebuild with a `grunt dev` and confirm the plugin and line of code that is throwing this error?

**Comment by chrisgillison** [comment]
Not done that before so not sure what I'm looking for! Have reun `grunt dev`, it ran through a bunch of things but now it seems to have stopped at Running "watch" task Waiting... Do I need to do something else?!

**Comment by chrisgillison** [comment]
Ah ok, I've reloaded the course, I'm getting this in the console:
`Uncaught (in promise) TypeError: firstQuestionBlock is undefined
    hideFutureBlocks adapt-block-branchingArticleModel.js:43
    Backbone 4
    _callee2$ adaptView.js:71
    f regenerator-runtime.min.js:7
    _invoke regenerator-runtime.min.js:7
    E regenerator-runtime.min.js:7
    asyncGeneratorStep adapt.min.js:34
    _next adapt.min.js:56
adapt-block-branchingArticleModel.js:43:21
`

**Comment by tomgreenfield** [comment]
It would seem your error is originating from a different plugin: [adapt-block-branching](https://github.com/simondate/adapt-block-branching).

**Comment by chrisgillison** [comment]
I got rid of adapt-block-branching. The errors disappeared but abs was still floopy. Created an entirely new course (FW 5.15.0) including v5 of abs and the latest version of  adapt-block-branching (1.0.1) and all is working fine.

The problem stemmed from the version of abs I was using which was an amalgam of the v5 version, and [the _slideCompletionLock version](https://github.com/cgkineo/adapt-articleBlockSlider/pull/45/files). I really wanted _slideCompletionLock, and I noticed the abs v5 version didn't have that functionality yet, so I updated the adapt-articleView.js in the abs v5 js folder to include it.

Just wanted to clarify  - that it was my (rather than Simon Date's) work that was the issue!
